libraries/AP_HAL_Linux/RCInput_SBUS.cpp

```diff
@@ -14,9 +14,9 @@
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 /*
-  this is a driver for DSM input in the QFLIGHT board. It could be
-  extended to other boards in future by providing an open/read/write
-  abstraction
+  this is a driver for SBUS input in Linux board using a UART. Note
+  that it relies on kernel support for 100kbaud and on a uart inverted
+  in hardware
```

Comment:
SBUS is inverted. DSM and SUMD aren't. 
On Disco they have two uarts, one inverted, one not, both connected to the same external pin. I want to try reading from both and see which one decodes to a valid RC input